conduit/init_provider.go

```diff
@@ -0,0 +1,26 @@
+package conduit
+
+import (
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+)
+
+// AlgodInitProvider algod based init provider
+type AlgodInitProvider struct {
```

Comment:
I don't understand which part of this is specific to Algod

Reply:
This would essentially have a client handle to algod since Will told me that most plugin developers won't have a location for the genesis file.  So here we would simply grab the genesis file from an algod instance.

Reply:
In that case I think we should just integrate this into the Importer plugin. Wherever we're fetching blocks from is going to have to have the genesis info so I think it makes sense to require it to be able to provide that as well as a block for a given round.